@afurs : A think your recent PR broke the MacOS builds. Could you please take a look and fix asap? 
```
System/Volumes/Data/build/ci_checks/alice_ci_0/o2/sw/SOURCES/O2/6562/0/Detectors/FIT/raw/include/FITRaw/DigitBlockFIT.h:91:19: error: implicit truncation from 'int' to bit-field changes value from 131071 to -1 [-Werror,-Wbitfield-constant-conversion]
    tcmData.amplA = 131071; //2^17
                  ^ ~~~~~~
/System/Volumes/Data/build/ci_checks/alice_ci_0/o2/sw/SOURCES/O2/6562/0/Detectors/FIT/raw/include/FITRaw/DigitBlockFIT.h:292:26: note: in instantiation of function template specialization 'o2::fit::DigitBlockFIThelper::ConvertDigit2TCMData<o2::ft0::Digit, o2::fit::TCMdata>' requested here
--
In file included from /System/Volumes/Data/build/ci_checks/alice_ci_0/o2/sw/SOURCES/O2/6562/0/Detectors/FIT/FT0/simulation/src/digit2raw.cxx:26:
In file included from /System/Volumes/Data/build/ci_checks/alice_ci_0/o2/sw/SOURCES/O2/6562/0/Detectors/FIT/FT0/raw/include/FT0Raw/RawWriterFT0.h:20:
In file included from /System/Volumes/Data/build/ci_checks/alice_ci_0/o2/sw/SOURCES/O2/6562/0/Detectors/FIT/FT0/raw/include/FT0Raw/DigitBlockFT0.h:19:
/System/Volumes/Data/build/ci_checks/alice_ci_0/o2/sw/SOURCES/O2/6562/0/Detectors/FIT/raw/include/FITRaw/DigitBlockFIT.h:96:19: error: implicit truncation from 'int' to bit-field changes value from 131071 to -1 [-Werror,-Wbitfield-constant-conversion]
    tcmData.amplC = 131071; //2^17
```

--- a/Detectors/Upgrades/ALICE3/TRK/simulation/src/Detector.cxx
+++ b/Detectors/Upgrades/ALICE3/TRK/simulation/src/Detector.cxx
@@ -126,7 +126,6 @@ void Detector::configITS(Detector* its)
 
   std::array<float, 12> sensorThicknesses = {100.e-4, 100.e-4, 100.e-4, 100.e-4, 100.e-3, 100.e-3, 100.e-3, 100.e-3, 100.e-3, 100.e-3, 100.e-3, 100.e-3};
   its->setStaveModelOB(o2::trk::Detector::kOBModel2);
-  // its->createOuterBarrel(false);
 
   auto idLayer{0};
   for (auto& layerData : layers) {

